### DISCUSSION TITLE
Frontend

### DISCUSSION THREAD

**Discussion by nikhil-agrawal123** [IIIT-Delhi/smp-portal#175] [General]
During the project local setup there is no folder name frontend to initiate the frontend

**Comment by Rezanmonis** [comment]
We are looking for a frontend developer as our teammate for our edtech project rezanmonis@gmail.com

**Reply by nikhil-agrawal123** [reply]
Can you share more details and the tech stack you require

**Comment by Rezanmonis** [comment]
Call me please +90 505 8562002

2 Eyl 2024 Pzt 08:06 tarihinde Nikhil agrawal ***@***.***>
şunu yazdı:
